tests/test_post_send.rs

```diff
@@ -126,6 +140,19 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             let slice = std::slice::from_raw_parts(mr.buf.data.as_ptr(), mr.buf.len);
             println!("Buffer contents: {:?}", slice);
         }
+
+        // poll recv CQ for the completion
```

Comment:
Would you mind add `post_recv` test for `test_ibv_wr`? Which tests extended QP

Reply:
Sure. BTW, since these tests have a lot of duplicated code, we may need to reorganize them in the future.